run.sh

```diff
@@ -13,6 +13,44 @@ if [ -z "$SUMO_ACCESS_ID" ] || [ -z "$SUMO_ACCESS_KEY" ]; then
 	exit 1
 fi
 
+# Support using env as replacement within sources.
+# Gather all template files
+declare -a TEMPLATE_FILES
+if [ -r "${SUMO_SOURCES_JSON}.tmpl" ]; then
+    TEMPLATE_FILES+=("${SUMO_SOURCES_JSON}.tmpl")
+fi
+if [ -r "${SUMO_SYNC_SOURCES}.tmpl" ]; then
```

Comment:
There may be some misunderstanding? SUMO_SYNC_SOURCES is a boolean value which indicate the source setting is controlled by local file or web config. It's not a file name. So we only need process one file here